docs/dev/annual_updates.rst

```diff
@@ -192,32 +202,63 @@ descriptive strings associated with each row in the FERC Form 1, and also indica
+
+* Consider whether the column could benefit from any of the standard transforms in
+  :mod:`pudl.transform.classes`. If so, add those transforms/the new column to
+  :py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS`. If the standard transform
+  you're applying to the new column was not previously listed in
+  :py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` under the table you're
+  working on, you'll need to add it to ``transform_main()`` in
+  :mod:`pudl.transform.ferc1` under the cooresponding table transformer class or it
+  won't be executed.
+
+* Consider whether the column could benefit from custom transformations. If it's
+  something that could be applicable to other columns, consider building it in
+  :mod:`pudl.tranform.classes`. If not, build it in the relevant table transformer
+  class in :mod:`pudl.transform.ferc1`.
+
+**4.B.4)** If there's a new table, add it to the transform process. You'll need to build
+or augment a table transformer in :mod:`pudl.transform.ferc1` and follow all
+instructions applicable to new columns.
+
+5. Update the PUDL DB Schema
 ----------------------------
-**5.1)** If new columns or tables have been added, you will need to update the
-PUDL DB schema, defining column types, giving them meaningful descriptions, applying
-appropriate ENUM constraints, etc. This happens in the :mod:`pudl.metadata` subpackage.
-Otherwise when the system tries to write dataframes into SQLite, it will fail.
-
-**5.2)** Differentiate between columns which should be harvested from the transformed
+If new columns or tables have been added, you must also update the PUDL DB schema,
+define column types, give them meaningful descriptions, apply appropriate ENUM
+constraints, etc. This happens in the :mod:`pudl.metadata` subpackage. Otherwise when
+the system tries to write dataframes into SQLite, it will fail or simply exclude any new
+columns.
+
+**5.1)** Check whether new columns exist in
+:py:const:`pudl.metadata.fields.FIELD_METADATA`. If they do, make sure the descriptions
+and data types match. If the descriptions don't match, you may need to define that
+column by source: :py:const:`pudl.metadata.fields.FIELD_METADATA_BY_GROUP` or by table:
+:py:const:`pudl.metadata.fields.FIELD_METADATA_BY_RESOURCE`. If the column is not in
+:py:const:`pudl.metadata.fields.FIELD_METADATA`, add it.
+
+**5.2)** Add new columns and tables to the ``RESOURCE_METADATA`` dictionaries in the
+appropriate :mod:`pudl.metadata.resources` modules.
+
+**5.3)** Update any :mod:`pudl.metadata.codes`, :mod:`pudl.metadata.labels`, or
+:mod:`pudl.metadata.enums` pertaining to new or existing columns with novel content.
+
+**5.4)** Differentiate between columns which should be harvested from the transformed
 dataframes in the normalization and entity resolution process (and associated with a
 generator, boiler, plant, utility, or balancing authority entity), and those that should
 remain in the table where they are reported.
```

Comment:
I don't think this is applicable to FERC, but I also don't really understand what this is saying. I know it relates to the harvesting process, but I don't understand what it's telling you to do.